### PR TITLE
Update it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -43,131 +44,143 @@ msgstr ""
...
-msgstr ""
+msgstr "Questa è una versione minore con alcune modifiche."
 
 #: data/org.parlatype.Parlatype.appdata.xml.in:55
 msgid ""
 "Automatic speech recognition (ASR) is now disabled by default and it’s only "
 "shown, if parlatype is launched with the new parameter --with-asr"
 msgstr ""
+"Il riconoscimento vocale automatico (ASR) è ora disabilitato per impostazione predefinita ed è solo"
+"mostrato, se parlatype viene lanciato con il nuovo parametro --con-asr"
```
Comment (gkarsay) [inline po/it.po]: Change --con-asr to --with-asr, this should not be translated

File: po/it.po [hunk 2]
```diff
@@ -224,100 +238,108 @@ msgstr ""
...
 #: src/pt-asr-assistant.c:194
 msgid "The folder contains no acoustic model."
-msgstr ""
+msgstr "La cartella non contiene alcun modello acustico"
 
 #. Translators: This feature is disabled by default, its translation has low priority.
 #: src/pt-asr-assistant.c:301
 msgid "Please select …"
-msgstr ""
+msgstr "Seleziona..."
```
Comment (gkarsay) [inline po/it.po]: Use the same ellipsis character as in the English string, it's a single Unicode character vs. 3 dots/characters

File: po/it.po [hunk 3]
```diff
@@ -342,21 +364,21 @@ msgstr[1] "secondi"
...
 #: src/pt-preferences.c:258
 msgid "Deleting configuration"
-msgstr ""
+msgstr "Eliminazione della configurazione"
 
 #: src/pt-preferences.c:259
 #, c-format
 msgid "Do you really want to delete “%s”?"
-msgstr ""
+msgstr "Vuoi veramente cancellare “%s”?"
```
Comment (gkarsay) [inline po/it.po]: Use \\" as quotation marks, like the Italian GNOME translation team: that would be "Vuoi veramente cancellare \\"%s\\"?"

File: po/it.po [hunk 4]
```diff
@@ -342,21 +364,21 @@ msgstr[1] "secondi"
...
 #: src/pt-window.c:217
 #, c-format
 msgid "Found %s"
-msgstr ""
+msgstr "Trovato %s"
 
 #: src/pt-window.c:218
 msgid "Press “Play” to start automatic speech recognition."
-msgstr ""
+msgstr "Premere "Riproduci" per avviare il riconoscimento vocale automatico."
```
Comment (gkarsay) [inline po/it.po]: \\" quotation marks around Riproduci

File: po/it.po [hunk 5]
```diff
@@ -403,6 +425,9 @@ msgid ""
 "GStreamer and make sure you have the “Good Plugins” installed.\n"
 "Parlatype will quit now, it received this error message: %s"
 msgstr ""
+"Per eseguire Parlatype è necessario GStreamer 1.x. Controlla l'installazione di "
+"GStreamer e assicurati di avere installato "Good Plugin".\n"
```
Comment (gkarsay) [inline po/it.po]: \" quotation marks

File: po/it.po [hunk 6]
```diff
@@ -505,112 +539,112 @@ msgstr "Forma d’onda"
 
 #: src/resources/help-overlay.ui:47
 msgid "Play/Pause"
-msgstr ""
+msgstr "Rproduci/Pausa"
```
Comment (gkarsay) [inline po/it.po]: Typo?

File: po/it.po [hunk 7]
```diff
@@ -505,112 +539,112 @@ msgstr "Forma d’onda"
...
 
 #: src/resources/help-overlay.ui:93
 msgid "Zoom out"
-msgstr ""
+msgstr "Zoom indietro"
 
 #: src/resources/help-overlay.ui:100
 msgid "Open “Go to” dialog"
-msgstr ""
+msgstr "Apri la finestra di dialogo "Vai a"
```
Comment (gkarsay) [inline po/it.po]: "Apri la finestra di dialogo \\"Vai a\\""

File: po/it.po [hunk 8]
```diff
@@ -505,112 +539,112 @@ msgstr "Forma d’onda"
...
 
 #: src/resources/preferences.ui:86
 msgid "Full seconds"
-msgstr ""
+msgstr "Secondi completi"
 
 #: src/resources/preferences.ui:90
 msgid "10th seconds"
-msgstr ""
+msgstr ""10 secondi""
```
Comment (gkarsay) [inline po/it.po]: Too many quotation marks

File: po/it.po [hunk 9]
```diff
@@ -644,67 +678,67 @@ msgstr "Cursore _fisso"
 
 #: src/resources/preferences.ui:288
 msgid "Jumps"
-msgstr ""
+msgstr "Salti"
 
 #: src/resources/preferences.ui:390
 msgid "Jump _forward:"
-msgstr "_Avanti:"
+msgstr "Salta_avanti:"
```
Comment (gkarsay) [inline po/it.po]: Space after Salta

File: po/it.po [hunk 10]
```diff
@@ -644,67 +678,67 @@ msgstr "Cursore _fisso"
...
+msgstr "Salta _indietro:"
 
 #: src/resources/preferences.ui:418
 msgid "Jump back on _pause:"
 msgstr "Indietro in _pausa:"
 
 #: src/resources/preferences.ui:441
 msgid "Pressing “Play” at the end of file"
-msgstr ""
+msgstr "Premendo "Riproduci" alla fine del file"
```
Comment (gkarsay) [inline po/it.po]: \\" quotation marks

File: po/it.po [hunk 11]
```diff
@@ -644,67 +678,67 @@ msgstr "Cursore _fisso"
...
 
 #: src/resources/preferences.ui:455
 msgid "Replay file"
-msgstr ""
+msgstr "Ripeti file"
 
 #: src/resources/preferences.ui:474
 msgid "Pressing “Play” at the end of selection"
-msgstr ""
+msgstr "Premendo "Riproduci" alla fine della selezione"
```
Comment (gkarsay) [inline po/it.po]: \\" quotation marks

File: po/it.po [hunk 12]
```diff
@@ -713,44 +747,44 @@ msgstr ""
...
+msgstr "Ricorda _dimensione"
 
 #: src/resources/preferences.ui:1038
 msgid "Remember _position"
 msgstr "Ricorda la _posizione"
 
 #: src/resources/preferences.ui:1053
 msgid "Start on _top"
-msgstr ""
+msgstr "inizia in_cima"
```
Comment (gkarsay) [inline po/it.po]: Start with capital letter; space after in

**Comment by gkarsay** [comment]
Mille grazie! There are a few things to change, please do that. I prefer when translations are done in Launchpad, but I will merge your work after the changes.

**Comment by albanobattistella** [comment]
> Mille grazie! There are a few things to change, please do that. I prefer when translations are done in Launchpad, but I will merge your work after the changes.

I'll fix problem.
thanks!

**Comment by gkarsay** [comment]
You have changed a few things, not everything though. Could you please go through the conversation list, reply to each conversation (e.g. Done or why not) and click on "Resolve conversation".

**Comment by gkarsay** [comment]
I committed your translation separately as 0e6cfc198492462553dfc863ae0fd33e7e019922 (changed commit messaged and squashed your 2 commits). Although this merge request is closed, your work is committed. Thank you!

**Pull request closed without merging**